RELEASE_NOTES.md

```diff
@@ -9,6 +9,20 @@
 - The minimum supported version of `protobuf` was bumped to 5.28.1 and to 1.66.1 for `grpcio`. Make sure to update your dependencies accordingly.
 - The `AddComponentBoundsRequest.validity_duration` field was replaced by `AddComponentBoundsRequest.request_lifetime` to match the fields in `SetComponentPowerActiveRequest` and `SetComponentPowerReactiveRequest`. The default value was also changed from 5 to 60 seconds.
 - The `AddComponentBoundsResponse.ts` field was renamed to `AddComponentBoundsResponse.valid_until` for extra clarity and also to match the fields in `SetComponentPowerActiveResponse` and `SetComponentPowerReactiveResponse`.
+- The version of `frequenz-api-common` has been updated to v0.7.0. Please refer to the [release notes](https://github.com/frequenz-floss/frequenz-api-common/releases/tag/v0.7.0) and update your code accordingly.
+  - Sensor categories have been removed from `frequenz-api-common`, and are therefore removed from the Microgrid API as well.
+  - `component` and `Component` have been renamed to `electrical_component` and `ElectricalComponent`, respectively. This is keep in line with the naming conventions used in `frequenz-api-common`.
+  - `ListComponentConnectionsResponse.connections` has been renamed to `ListElectricalComponentConnectionsResponse.electrical_component_connections` to match the naming conventions used in `frequenz-api-common`.
+  - `ReceiveComponentDataStreamResponse.data` has been renamed to `ReceiveElectricalComponentDataStreamResponse.telemetry` to match the naming conventions used in `frequenz-api-common`.
+  - `ReceiveSensorDataStreamResponse.data` has been renamed to `ReceiveSensorDataStreamResponse.telemetry` to match the naming conventions used in `frequenz-api-common`.
+  - `ListComponentConnectionsRequest.starts` has been renamed to `ListElectricalComponentConnectionsRequest.from_electrical_component_id` to match the naming conventions used in `frequenz-api-common`.
+  - `ListComponentConnectionsRequest.ends` has been renamed to `ListElectricalComponentConnectionsRequest.to_electrical_component_id` to match the naming conventions used in `frequenz-api-common`.
```

Comment:
Nitpick: What about `source`/`destination_electrical_component_id` as proposed in https://github.com/frequenz-floss/frequenz-api-microgrid/issues/262? The new version of the client is already using `source` and `destination` (or course I can change it too).

But when reading the "from/to" it felt like the meaning was more like "from electrical component to sensor" or something like that, it felt like "from" makes "electrical component" highlight more and expect a different "to" while with "source electrical component" makes my brain think then I should expect a destination of the same type (electrical component).

But also if you are talking about properties of a connection, you can talk about a connection source or destination, while a connection from or to sounds super weird.